### PR TITLE
Initial commit on material

### PR DESCRIPTION
**Pull request by billlee77**
### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [x ] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ x] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
No
### Does this PR change default behavior?
No

### CODE REVIEW

File: src/PFRICH_geo.cpp [hunk 1]
```diff
@@ -275,18 +286,34 @@ static Ref_t createDetector(Detector& description, xml_h e, SensitiveDetector se
...
+  /*--------------------------------------------------*/
+  ///        ag_name.Form("%s-%d-%02d", "aerogel", ir, ia);
+  ///
+  ///        Volume agtubeVol(ag_name.Data(), agtube, gasvolMat);
+  ///        auto aerogelTilePlacement = Transform3D(r_aerogel_Z, Position(0.0, 0.0, -m_gzOffset));
+  ///        auto aerogelTilePV        = pfRICH_volume.placeVolume(agtubeVol, aerogelTilePlacement);
+  ///        DetElement aerogelDE(sdet, "aerogel_de_" + std::to_string(kkcounter), 0);
+  ///        aerogelDE.setPlacement(aerogelTilePV);
+
+  //  Volume vesselVol(detName + "_bbbbbbbbbbbbbbbbbbbbbbbbbbb_vol", vesselSolid, vesselMat);
```
Comment (veprbl) [inline src/PFRICH_geo.cpp:301]:
```suggestion
```
Please don't add commented code to the shared repository.

File: src/PFRICH_geo.cpp [hunk 2]
```diff
@@ -275,18 +286,34 @@ static Ref_t createDetector(Detector& description, xml_h e, SensitiveDetector se
...
+  Volume vesselVol(detName + "_bbbbbbbbbbbbbbbbbbbbbbbbbbb_vol", wallSolid, vesselMat);
```
Comment (veprbl) [inline src/PFRICH_geo.cpp:302]: Please give a meaningful suffix here.

File: src/PFRICH_geo.cpp [hunk 3]
```diff
@@ -275,18 +286,34 @@ static Ref_t createDetector(Detector& description, xml_h e, SensitiveDetector se
...
-  DetElement gasvolDE(sdet, "gasvol_de", 0);
-  gasvolDE.setPlacement(gasvolPV);
+  //  Volume gasvolVol(detName + "_aaaaaaaaaaaaaaaaaaaaaaaaaaa_gas", gasvolSolid, gasvolMat);
+  //  gasvolVol.setVisAttributes(gasvolVis);
+  //
+  //  // place gas volume
+  //  // PlacedVolume gasvolPV = vesselVol.placeVolume(gasvolVol, Position(0, 0, 0));
+  //  PlacedVolume gasvolPV = pfRICH_volume.placeVolume(gasvolVol, Position(0, 0, 0));
+  //  DetElement gasvolDE(sdet, "gasvol_de", 0);
+  //  gasvolDE.setPlacement(gasvolPV);
```
Comment (veprbl) [inline src/PFRICH_geo.cpp:316]:
```suggestion
```
Same as above.

File: src/PFRICH_geo.cpp [hunk 4]
```diff
@@ -113,11 +111,16 @@ static Ref_t createDetector(Detector& description, xml_h e, SensitiveDetector se
...
   double airgapThickness = 0.1;
   double filterThickness = 1;
 
-  auto aerogelMat = description.material("C4F10_PFRICH");
-  auto filterMat  = description.material("C4F10_PFRICH");
+  auto aerogelMat = description.material(aerogelElem.attr<std::string>(_Unicode(material)));
+  auto filterMat  = description.material(filterElem.attr<std::string>(_Unicode(material)));
+
+  //  auto aerogelMat = description.material("C4F10_PFRICH");
+  //  auto filterMat  = description.material("C4F10_PFRICH");
```
Comment (veprbl) [inline src/PFRICH_geo.cpp:123]:
```suggestion
```

The change is recorded in the git history. You can use `git log -p -- src/PFRICH_geo.cpp` to access relevant changes.

**Review by veprbl — COMMENTED** [review]
Thank you for your contribution. I have code organization suggestions.

File: src/PFRICH_geo.cpp [hunk 5]
```diff
@@ -319,18 +321,28 @@ static Ref_t createDetector(Detector& description, xml_h e, SensitiveDetector se
...
+  auto HRPPD_WindowMat = description.material("Quartz");
+  auto HRPPD_PCBMat    = description.material("G10");
+  auto HRPPD_MPDMat    = description.material("SiliconDioxide");
+  auto HRPPD_ASICMat   = description.material("SiliconCarbide");
+
   Box hrppd_Solid(xysize / 2, xysize / 2, hrppd_container_volume_thickness / 2);
 
   Volume hrppdVol_air(detName + "_air_hrppd", hrppd_Solid, air);
-  Volume hrppdVol(detName + "_hrppd", hrppd_Solid, sensorMat);
+  //  Volume hrppdVol(detName + "_hrppd", hrppd_Solid, sensorMat);
```
Comment (veprbl) [inline src/PFRICH_geo.cpp:345]:
```suggestion
```

**Review by veprbl — APPROVED** [review]

**Comment by billlee77** [comment]
@wdconinc @kkauder

Hi Wouter and Kolja, the PR is ready for review and merge

**Pull request merged**